dist/accordion/css/accordion.min.css

```diff
@@ -1 +1,62 @@
-.aria-accordion__panel[hidden]{display:none}
\ No newline at end of file
+/* For legacy browsers support */
+.aria-accordion__panel[hidden] {
+  display: none;
+}
+
+/* For accordion panels with a transition effect */
+.aria-accordion__panel--transition {
+  display: block;
+  max-height: 100vh;
+  visibility: visible;
+  -webkit-transition: max-height 800ms ease-in;
+  -moz-transition: max-height 800ms ease-in;
+  -ms-transition: max-height 800ms ease-in;
+  -o-transition: max-height 800ms ease-in;
+  transition: max-height 800ms ease-in;
+  overflow: visible;
+  /* persist overflow value from animation */
+  animation: 800ms delay-overflow;
+  animation: 900ms delay-padding-expanding;
```

Comment:
```suggestion
```

Regarding `animation` CSS properties, it is very cool and powerful. However, I think that here (in the library context) as minimum required styles for transition we do not need it. You can use it for the CKEditor plug-in or on a per-project basis as additional styles.